c/src/ml-api-service-agent-client.c

```diff
@@ -17,6 +17,84 @@
 #include "ml-api-service.h"
 #include "ml-api-service-private.h"
 
+#if defined(__TIZEN__)
+#include <app_common.h>
+#endif
+
+#if defined(__TIZEN__)
```

Comment:
you can remove line 22~24, same feature.

Reply:
Thanks, I removed those lines. PTAL